### DISCUSSION TITLE
Remember window size and position?

### DISCUSSION THREAD

**Discussion by gerroon** [LazyVim/LazyVim#2012] [Q&A] [answered]
Hi

I am using LV with GoNeovim on Win 11.  I am wondering if there is a way to make it remember the last window size/pos or somekind of  better scaled windowed? Would that be LV or Neovim thing? I have a 4k monitors and Goneovim opens quite small everytime I open it.

**Comment by lvl4ul2i** [comment, marked as answer]
Hi @gerroon , I don't know if it helps, I don't have a 4k monitor but a 32 ultrawide and I was having problems with the windows. I tried [windows](https://github.com/anuvyklack/windows.nvim) plugin and it worked fine. You can set a minimum width, and the animations are very cool.

**Comment by dpetka2001** [comment]
That would most likely seem a problem with how your Desktop Environment treats your applications. You can check this [post](https://www.elevenforum.com/t/how-to-default-all-windows-to-maximized.7405/#post-150325) for how to make Windows 11 start your apps maximized.

**Reply by gerroon** [reply]
Thanks for the suggestion, I do not want maximized, I just need it to open with a reasonable size.

**Reply by dpetka2001** [reply]
That's up to the Desktop Environment as I've said. I don't use Windows, so I can't be much more of help. Plz try to google about "Windows 11 remember window positions" for clues as how to achieve what you want.

**Reply by gerroon** [reply]
Windows remembers window positions and sizes, there is no problem with it, at least for some programs. Goneovim opens right in the middle as like 600x600 (out of 4k) kind  of size, I assumed there was a default with LV that makes it open this way.

**Reply by dpetka2001** [reply]
Nope. Nothing like that exists in LazyVim. LazyVim is just a plugin for Neovim. It doesn't define the dimensions of how Neovim is opened. That is up to the Desktop Environment to decide (and whatever frontend client is used for Neovim, in this case `GoNeovim`). Maybe make an issue at `GoNeovim`'s repo, as they should have a better idea for how their app should behave on Windows 11?

**Reply by gerroon** [reply]
Thanks, I will pursue it with Goneovim.
